commission/wizards/commission_make_settle.py

```diff
@@ -97,7 +103,7 @@ def _prepare_settlement_line_vals(self, settlement, line):
             "settlement_id": settlement.id,
         }
 
-    def _get_agent_lines(self, date_to_agent):
+    def _get_agent_lines(self, agent, date_to_agent):
```

Comment:
Why do you need to add this argument? This is breaking possible extensions.

Reply:
I do not know... 
All modules that extends this method are changing the signature... So I change the signature in original method

Reply:
Ah, really? Then it can be considered a bug. Can you link one of those overrides?

Reply:
https://github.com/OCA/commission/blob/3a55b78b7f4773771577dc83d12d129a08147be1/account_commission/wizards/commission_make_settle.py#L17

Reply:
Then it's definitively a bug here, so it's OK to change it indeed, thanks.